### PR TITLE
Fix #2318 - breaking changes page

### LINKED ISSUE #2318: Breaking changes / install diff file for major version

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a page to the docs.

**How to test**:
1. open page and confirm readable

**Expected outcome**:

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: docs/admin-guide/breaking.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from scratch when a new release is out. Essentially always it is safe to go to a new
+patch or minor version with your current setup. This is one of the beautiful things with
+using a document database and REST interfaces. Occasionaly, especially when interaction
+with other components change, a few steps will have to be taken. We will try to detail
+them here.
+
+A quick revert to an older version is mostly possible. There are however often foreseen
+or unforeseen issues with reverting to older release versions,
+especially if external data has been loaded or cases and events have been added. We try not to
+make too much of an impact, and have often reverted a few times in a stage environment,
```
Comment (northwestwitch) [inline docs/admin-guide/breaking.md:13]: often or few times?

**Review by northwestwitch — APPROVED** [review]
Nice! 👍

File: docs/admin-guide/breaking.md (hunk 1 above)
Comment (dnil) [reply docs/admin-guide/breaking.md:13]: Both, actually, but let me rephrase!

**Comment by dnil** [comment]
`_`and `-`appears interchangeable on pip. Nice!
